vagrant.go

```diff
@@ -96,7 +96,8 @@ type Vagrant struct {
 func (v *Vagrant) setup(start bool, env []string, numNodes int) error {
 	v.nodes = map[string]TestbedNode{}
 
-	vCmd := &VagrantCommand{ContivNodes: numNodes}
+        e:=strings.Join(env," ")
```

Comment:
I worry this will break with any environment variables that have space in the values.